node/block.go

```diff
@@ -18,7 +18,7 @@ func (node *Node) AddNewBlock(ctx context.Context, b *types.Block) (err error) {
 	// Put block in storage wired to an exchange so this node and other
 	// nodes can fetch it.
 	log.Debugf("putting block in bitswap exchange: %s", b.Cid().String())
-	blkCid, err := node.OnlineStore.Put(ctx, b)
```

Comment:
wait, i thought the cborstore was for state tree?

Reply:
I would gladly change it to call into the node's blockstore.  I could also (1) add a method to the fetcher for local additions to its blockstore (2) change the syncer interface to accept blocks directly and stop being indirect like this.

Reply:
seems reasonable